decidim-core/db/migrate/20161020080756_add_position_to_steps.rb

```diff
@@ -0,0 +1,5 @@
+class AddPositionToSteps < ActiveRecord::Migration[5.0]
+  def change
+    add_column :decidim_participatory_process_steps, :position, :integer
```

Comment:
Agreed offline to just add a `before_create` action to set the value when it's `nil`.